sensor/kubernetes/eventpipeline/pipeline_impl.go

```diff
@@ -61,6 +61,8 @@ func (p *eventPipeline) ProcessMessage(msg *central.MsgToSensor) error {
 		return p.processReprocessDeployment(msg.GetReprocessDeployment())
 	case msg.GetInvalidateImageCache() != nil:
 		return p.processInvalidateImageCache(msg.GetInvalidateImageCache())
+	case msg.GetPong() != nil:
+		log.Infof("Received Pong message from Central.")
```

Comment:
this could potentially be overwhelming if we print this every 30 seconds. Maybe a debug log?

Reply:
I see this is also logged in ProcessMessage. Is this meant to be a call to ProcessMessage?

Reply:
Ah this should have been removed, since `ProcessMessage` will be invoked by the central receiver.